core/utils.py

```diff
@@ -1179,3 +1182,18 @@ def quoted(s: str) -> str:
         str. The quoted string.
     """
     return json.dumps(s)
+
+
+def url_open(source_url: str) -> Any:
```

Comment:
Why return `Any`?

Reply:
yes @vojtechjelinek i also have doubt in this,
i try urlopen object, cappable etc.

but every time mypy tests give error and indicate to use Any.
??

Reply:
I think it should be `HTTPResponse`, but you will need to investigate from where to import it.

Reply:
![Screenshot from 2021-11-26 02-33-55](https://user-images.githubusercontent.com/80846488/143499790-58f89c5e-3a30-4ded-9f73-2c622535214e.png)

Same no difference.

Reply:
Can you try `reveal_type(urlrequest.urlopen(source_url, context=context))`, then run the mypy checks and send me the output?

Reply:
![Screenshot from 2021-11-28 21-52-05](https://user-images.githubusercontent.com/80846488/143776784-80d368c4-10a4-439c-8895-58c88be7c15f.png)

Reply:
Okay, thanks, left some other comments that should fix this.

Reply:
Done